main.js

```diff
@@ -1 +1,49 @@
-console.log('hello word');
+let user;
+user = 'John Doe';
+console.log(user);
+
+let student;
+// eslint-disable-next-line prefer-const
+student = 'Daria Ramanowska';
+console.log(student);
+
+user = student; //переопределены данные на 'Daria Ramanowska'
+console.log(user);
+
+let test = 1;
+test++;
+
+console.log(test + '1'); // '21' string
+console.log(test - 1); // 1 number
+console.log(test === 3); //false
```

Comment:
It doesn't assign with the task requirement. You should convert the value of variable `test` to a boolean.